overlord/devicestate/users.go

```diff
@@ -74,7 +74,7 @@ func CreateUser(st *state.State, sudoer bool, email string) (*CreatedUser, error
 	}
 
 	opts.Sudoer = sudoer
-	return addUser(st, username, email, opts)
+	return addUser(st, username, email, expiration, opts)
```

Comment:
the passing of expiration here is not tested, if I remove it nothing fails

Reply:
Nice catch! This is now covered